ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -806,8 +806,30 @@ objects:
         if [[ "${ENABLE_PROXY}" == "true" ]]; then
           cat >> /tmp/artifacts/installer/install-config.yaml << EOF
         proxy:
-          httpsProxy: http://admin:admin@35.196.128.173:3128/
+          httpsProxy: https://admin:admin@35.231.5.161:3128/
           httpProxy: http://admin:admin@35.196.128.173:3128/
+          additionalTrustBundle: |
```

Comment:
@ewolinetz `additionalTrustBundle` is not a `proxy` field, so get rid of the indentation.